weltyt.py

```diff
@@ -139,34 +139,39 @@ def buy_lands(lands):
         "growth_rate": 5,
         "price_ISL": 500,
         "plants": 0,
+        "growth_percentage": 0,
```

Comment:
Is it needed here at all? (the same for `plants`)

Reply:
Yeah, but while buying we update db only, not this dictionary.
In fact, this dictionary never changes, am I right?